cloud-init.yml

```
@@ -195,7 +199,7 @@ write_files:
     permissions: '0775'
     content: |
       #!/bin/bash
-
+      set -exu -o pipefail
```

```suggestion
      set -eu -o pipefail
```
This is a cron job, so shouldn't produce needless output.